core/proto/core.proto

```diff
@@ -43,6 +43,19 @@ message Certificate {
   optional int64 expires = 6; // Unix timestamp (nanoseconds)
 }
 
+message CertificateStatus {
```

Comment:
One question that I don't really know the answer to: how does grpc handle the case where I send a message from one service where the proto was defined in one place to a service where the proto has been moved to somewhere else? I'd think it should handle it just fine, but I worry that there might be some magic bytes different somewhere that could be a deployability issue?

Reply:
As long as the protos have the same fields, it doesn't care! (This comes "for free" along with proto's backwards compatibility: if someone sends you a proto with extra fields that you don't recognize, it'll still parse the parts it does recognize just fine. Since that necessarily means you're not using byte-for-byte compatible definitions of the protos, it works for this case too.)